create.py

```diff
@@ -1,7 +1,16 @@
 from cal import app
-from cal.schema import db
+from cal.schema import db, User
```

Comment:
Could we import these just from `cal` instead of `cal.schema`? I think having a central location for imports would be better (less coupling between modules)